packages/api/README.md

```diff
@@ -136,3 +136,55 @@ We use [pickup](https://github.com/web3-storage/pickup) to fetch DAGs from IPFS
 - `PICKUP_BASIC_AUTH_TOKEN` must be set as a secret in the env.
 
 For local dev, we use a local ipfs-cluster container for the same service.
+
+## w3up
+
+Some uploads sent to nft.storage/api will be sent to up.web3.storage (aka 'w3up') for storage, serving on IPFS, and persistence to filecoin.
+
+All uploads sent to w3up will be stored in the same web3.storage space configured by env var `W3_NFTSTORAGE_SPACE`.
+
+### using console.web3.storage to browse uploads to w3up
+
+You can use console.web3.storage to browse uploads in the W3_NFTSTORAGE_SPACE.
+
+The DID used by your console.web3.storage session will need to be authorized to access the space.
+
+The credentials used in staging/production are in the usual vault of secrets under 'w3up credentials'.
+
+Run the cli command `w3up console ucan generate`
+
+```shell
+(
+  cd packages/api
+  node scripts/cli.js w3up console ucan generate
```

Comment:
Oh lol but I just noticed this - also seems fine. Since this README is already in the `packages/api` directory it feels a little weird to have that here BUT it's unambiguous, where omitting it wouldn't be. Overall I think you could go either way, so maybe just ignore this and the previous comment ;-p